backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -69,40 +133,54 @@ nav.menu {
 
     &.selected {
       background: $color-navbar-active-bg;
+      @include media($desktop) {
+        position: relative;
+      }
 
-      > a {
-        color: $color-navbar-active;
+      &:not(:hover) > ul {
+        @include media($desktop) {
+          display: none;
+        }
+        @include media($tablet) {
+          display: block;
+        }
       }
-    }
 
-    &.selected li {
-      padding-left: 2em;
-    }
+      > ul {
+        @include media($desktop) {
+          @include flyout;
```

Comment:
The flyout is triggered by hovering, so it doesn't actually work on a touch screen. it's more of a bonus for desktop users than a feature removed for mobile though. With a single tap they can still open the menu to access the lower levels.